### ISSUE TITLE
About Dataset

### ISSUE THREAD

**Issue by xuxuxuxuxuxjh** [JCruan519/EGE-UNet#7]
I would like to ask, how is the dataset processed?
Is it true that the official data's training set and validation set are merged and then divided into an experimental training set and test set with a 7:3 ratio?
